crates/matrix-sdk-crypto/src/verification/cache.rs

```diff
@@ -158,7 +158,7 @@ impl VerificationCache {
     }
 
     pub fn add_request(&self, request: OutgoingRequest) {
-        trace!("Adding an outgoing verification request {:?}", request);
+        trace!("Adding an outgoing request {:?}", request);
```

Comment:
it's not always a verification request (sometimes it is a key share), so this was a bit confusing